google-datacatalog-kafka-connector/src/google/datacatalog_connectors/kafka/config/tag_template_constants.py

```diff
@@ -93,8 +93,11 @@ def __init__(self):
             self.consumer_groups = TagTemplateField(
                 'consumer_groups', 'Consumer groups',
                 datacatalog_v1beta1.enums.FieldType.PrimitiveType.STRING)
-            self.schema = TagTemplateField(
-                'schema', 'Schema',
+            self.key_schema = TagTemplateField(
```

Comment:
Can we name those 'physical_schema' ? To match the name the field will have in DC, and to show that this is different from 'logical' schema